ZenPacks/zenoss/ZenPackLib/lib/params/EventClassSpecParams.py

```diff
@@ -1,6 +1,6 @@
 ##############################################################################
 #
-# Copyright (C) Zenoss, Inc. 2016, all rights reserved.
+# Copyright (C) Zenoss, Inc. 2017, all rights reserved.
```

Comment:
Could you make the copyright 2016-2017 instead of removing the 2016? I think it's more important to include the original date than it is to include new dates.